src/backends/odbc/statement.cpp

```diff
@@ -352,6 +352,17 @@ void odbc_statement_backend::describe_column(int colNum, data_type & type,
     char const *name = reinterpret_cast<char const *>(colNameBuffer);
     columnName.assign(name, std::strlen(name));
 
+    SQLLEN is_unsigned = 0;
+    SQLRETURN rc_colattr = SQLColAttribute(hstmt_, static_cast<SQLUSMALLINT>(colNum),
+                                           SQL_DESC_UNSIGNED, 0, 0, 0, &is_unsigned);
+
+    if (is_odbc_error(rc_colattr))
+    {
+        std::ostringstream ss;
+        ss << "getting \"unsigned\" col attribute of the column at position " << colNum;
```

Comment:
As this message could be shown to the user, it would be better to avoid abbreviations in it:
```suggestion
        ss << "getting \"unsigned\" column attribute of the column at position " << colNum;
```